bernedoodle/index.html

```diff
@@ -0,0 +1,170 @@
+      href="../assets/vendor/fontawesome/6.2.1/css/fontawesome.min.css"
+    />
+    <link
+      rel="stylesheet"
+      href="../assets/vendor/fontawesome/6.2.1/css/brands.min.css"
+    />
+    <link
+      rel="stylesheet"
+      href="../assets/vendor/fontawesome/6.2.1/css/solid.min.css"
+    />
+    <link rel="stylesheet" href="../assets/styles/main.css" />
+    <link
+      rel="shortcut icon"
+      href="../assets/images/favicon.ico"
+      type="image/x-icon"
+    />
+  </head>
+
+  <body>
+    <header>
+      <img src="../assets/images/header.png" alt="Doodle Dandy" />
+      <nav>
+        <a href="/">Home</a>
+        <a href="/bernedoodle">Bernedoodle Info</a>
+        <a href="/breed">AussieDoodle Info</a>
+        <a href="/about">About us</a>
+        <a href="/current" class="current">Current Litter</a>
+        <a href="/famstories">Reviews</a>
+        <a href="/contact">Contact us</a>
+      </nav>
+    </header>
+    <main>
+      <h1>Bernedoodle Information</h1>
+
+      <h3>Welcome to the Wonderful World of Bernedoodles!</h3>
+
+      <img
+        src="../assets/images/ddpup2.jpg"
+        alt="Doodle Dandy Puppy Love"
+        class="dog"
+      />
+      <h3>What is a Bernedoodle?</h3>
+      <p>
+        Are you looking for a loyal, intelligent, and affectionate companion to
+        join your family? Look no further than the delightful Bernedoodle! This
+        increasingly popular hybrid breed combines the best traits of two
+        beloved purebreds: the calm and majestic Bernese Mountain Dog and the
+        highly intelligent and adaptable Poodle. The result is a truly special
+        dog that brings joy, laughter, and endless snuggles into any home.
+      </p>
+
+      <img
+        src="../assets/images/blueeyedberne.jpeg"
+        alt="Doodle Dandy Puppy Love"
+        class="dog"
+      />
+      <p>
+        Depending on the size of the Poodle used (standard, miniature, or toy),
+        Bernedoodles can range in size from tiny companions to large, cuddly
+        giants. Their coats are typically wavy or curly and come in a variety of
+        colors, often showcasing the beautiful tri-color markings of the
+        Bernese. One of their most sought-after traits is their low-shedding
+        coat, making them a more manageable option for many families.
+      </p>
+      <p></p>
+      <img
+        src="../assets/images/ddpup3.jpg"
+        alt="Doodle Dandy Puppy Love"
+        class="dog"
+      />
+      <p>
+        They are known for their friendly, playful personalities and are often
+        described as great family dogs. Bernedoodles come in various sizes,
+        including standard, mini, and tiny, and can have different coat colors
+        and types.
+      </p>
+      <img
+        src="../assets/images/echobaby.png"
+        alt="Doodle Dandy Puppy Love"
+        class="dog"
+      />
+
+      <p>
+        Whether as a family pet or a therapy animal, bernedoodles inherit a
```

Comment:
be consistent with how you capitalize the breed names
```suggestion
        Whether as a family pet or a therapy animal, Bernedoodles inherit a
```

Reply:
done